weave/refs.py

```diff
@@ -0,0 +1,39 @@
+"""Support for a collection of refs."""
+
+from typing import Iterable, Optional
+
+from rich.table import Table
+from . import graph_client_context
+
+from weave.trace.refs import parse_uri, AnyRef, CallRef
+from weave.rich_container import AbstractRichContainer
+from weave.trace.vals import TraceObject
+
+
+class Refs(AbstractRichContainer[str]):
+    """A collection of ref strings with utilities."""
+
+    def __init__(self, refs: Optional[Iterable[str]] = None) -> None:
+        super().__init__("Ref", refs)
+
+    def _add_table_columns(self, table: Table) -> None:
+        table.add_column("Ref", overflow="fold")
+
+    def _item_to_row(self, item: str) -> list:
+        return [item]
+
+    def parsed(self) -> list[AnyRef]:
+        return [parse_uri(ref) for ref in self.items]
+
+    def call_refs(self) -> "Refs":
+        return Refs(ref for ref in self.items if isinstance(parse_uri(ref), CallRef))
+
+    # TODO: Perhaps there should be a Calls that extends AbstractRichContainer
+    def calls(self) -> list[TraceObject]:
```

Comment:
2 notes:
1. This could be optimized by grouping all the ids together then executing a single request instead of N network requests
2. I feel like this wants to be on `FeedbackQuery` itself, so that my code is:
```
client.feedback(...).calls()
```

Reply:
Thanks, can definitely address these in a followup.